PracticeGit/Cash.h

```diff
@@ -2,12 +2,13 @@
 // Training ~ A training program for new joiners at Metamation, Batch - July 2024.
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
-// Program.h
+// Cash.h
 // ------------------------------------------------------------------------------------------------
 
-#ifndef PROGRAM_H
-#define PROGRAM_H
+#ifndef CASH_H
+#define CASH_H
 
-void PrintMessage ();         
+/// <summary>Generate the changes for the given amount.</summary>

```

Comment:
When providing a desc for a function, you must document what it returns (especially if you have a special value like -1 being returned)